src/Entity/Admin.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\AdminRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @ORM\Entity(repositoryClass=AdminRepository::class)
+ */
+class Admin implements UserInterface
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=180, unique=true)
+     */
+    private $email;
+
+    /**
+     * @ORM\Column(type="json")
+     */
+    private $roles = [];
```

Comment:
Do we plan to have roles? I feel like this should be more "permissions" than roles?

And then we probably should have a "isSuperAdmin" field to have all permissions.

Reply:
Better remove it then because there's no need to handle permissions right now.
And I don't think it would be a problem to add it later if needed.

Reply:
Well, seems like roles on user is required by Symfony ?